### PR TITLE
cypress production workflow

### PR DESCRIPTION
**Pull request by ghost**
Breaking the cypress ci flow into its own file specifically to cover the production env.  
The workflow as-is is set to run when:
1.  you create a PR/push changes
2. cron scheduled to run on master/prod at 2AM, Mon-Fri

Note: The e2e tests run in CI do NOT currently test against PR changes. We will need to add a PR comment command that accepts our staging server urls as a param.

Testing notes:
- [x] Cypress defaults to development env locally, tests pass
- [x] Cypress e2e tests run and pass against prod in CI flow

After merge:
- [ ] Cypress e2e tests run Mon-Fri at 2AM; tests pass

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -1,7 +1,3 @@
 {
-  "env": {
-    "apiURL": "localhost:3000/.netlify/functions/server"
-  },
-  "baseUrl": "http://localhost:8000",
-  "chromeWebSecurity": false
+
```
Comment (dancingshell) [inline cypress.json:2]: do you still need this file at all?

**Review by dancingshell — APPROVED** [review]
this is cool but how can you get it to run on the newly spun up staging branch?

File: cypress.json (hunk 1 above)
Comment (ghost) [reply cypress.json:2]: I wanted to keep it around just in case we needed a global config, but yeah it's possible we don't even need it now

**Comment by ghost** [comment]
@dancingshell yeah I'm looking for a way to trigger a cypress e2e run against a newly spun up staging environment with like a pr comment or something. Hopefully gonna be the next thing I work on in a follow-up.

Actually, it looks like there's already a netlify/cypress integration that should be able to automate this nicely for us:
https://www.cypress.io/blog/2020/03/30/run-cypress-tests-on-netlify-using-a-single-line/

File: cypress.json (hunk 1 above)
Comment (ghost) [reply cypress.json:2]: nope it looks like removing this file breaks things

**Pull request merged**